front/src/components/boxs/device-in-room/device-features/sensor-value/AirQualityDeviceValue.jsx

```diff
@@ -0,0 +1,58 @@
+import { Text } from 'preact-i18n';
+import cx from 'classnames';
+
+import { DEVICE_FEATURE_CATEGORIES } from '../../../../../../../server/utils/constants';
+import RawDeviceValue from './RawDeviceValue';
+
+const getBadgeColor = value => {
+  if (value < 50) {
+    // Safe
+    return 'success';
+  } else if (value < 100) {
+    // Moderate
+    return 'warning';
+  } else if (value < 150) {
+    // Unhealthy for Sensitive Groups
+    return 'orange';
+  } else if (value < 200) {
+    // Unhealthy
+    return 'pink';
+  } else if (value < 300) {
+    // Very Unhealthy
+    return 'purple';
+  }
+  // Hazardous
+  return 'danger';
+};
+
+const BADGE_CATEGORIES = {
+  [DEVICE_FEATURE_CATEGORIES.AIRQUALITY_SENSOR]: value => getBadgeColor(value)
+};
+
+const AirQualityDeviceValue = props => {
```

Comment:
Why you don't use BadgeNumberDeviceValue?
You can add you own "colorMethod" for you category.

Reply:
I was thinking this feature is too specific and it would be more simple to maintain. I can use BadgeNumber if you prefer.

Reply:
Agree, this file is exactly like BadgeNumberDeviceValue

You can just use a custom getBadgeColor function specifically for air quality :)